src/bp/ui-studio/src/web/components/Layout/index.tsx

```diff
@@ -181,7 +190,19 @@ const Layout: FC<ILayoutProps & StateProps> = props => {
     'go-module-qna': () => gotoUrl('/modules/qna'),
     'go-module-testing': () => gotoUrl('/modules/testing'),
     'go-module-analytics': () => gotoUrl('/modules/analytics'),
-    'go-understanding': () => gotoUrl('/modules/nlu')
+    'go-understanding': () => gotoUrl('/modules/nlu'),
+    'zoom-in': e => {
+      e.preventDefault()
+      const event = new CustomEvent('zoomEvent', { detail: 'in' })
+
+      document.dispatchEvent(event)
+    },
+    'zoom-out': e => {
+      e.preventDefault()
+      const event = new CustomEvent('zoomEvent', { detail: 'out' })
+
+      document.dispatchEvent(event)
```

Comment:
Just curious why you are not using redux to update the zoom level. You could add two methods to the flows reducer (zoomIn and zoomOut) and keep the logic there instead